components/dialog/dialog.js

```diff
@@ -22,6 +22,10 @@ class Dialog extends LocalizeCoreElement(AsyncContainerMixin(DialogMixin(LitElem
 
 	static get properties() {
 		return {
+			/**
+			 * Whether to read the contents of the dialog on open
+			 */
+			ariaDescribeContent: { type: Boolean, attribute: 'aria-describe-content' },
```

Comment:
Agreed with everything Margaree mentioned here -- I like calling this `describe-content`.

Also it might be good to have some kind of guidance in the documentation for when and when not this should be used... like should we be defaulting it to `true` for all our prompt-style dialogs? And if so, maybe this doesn't need to be a customizable thing at all but just something that's built into the `<d2l-dialog-confirm>`?

Reply:
This is already built in to `d2l-dialog-confirm` but we need it in `d2l-dialog` as well. Not sure if it would make sense to build it in there as well unconditionally?